.buildkite/release/build.rayci.yml

```diff
@@ -22,6 +25,9 @@ steps:
           - cpu
 
   - label: ":tapioca: build: anyscale-ml py{{matrix}}-cu11.8.0 docker"
+    tags: skip-on-premerge
+    key: anyscalemlbuild
+    instance_type: release
```

Comment:
this is exposing anyscale image building on ray postmerge now, which does not feel quite right.

maybe we can just have a per commit build on release test pipeline? or maybe a `release-perf` pipeline?

Reply:
postmerge probably should also be private; the fact that someone can create a PR and run it on postmerge (through a new build) and postmerge has right access also does not seem right

Reply:
> someone can create a PR and run it on postmerge

only authorized members in the buildkite org can launch manual runs. but the results are public, so any one can see it.

Reply:
any strong opinion if I ask in #opensource to see if we can make it private; I'm leaning toward a direction of less number of pipelines if we can rather than more

Reply:
hmm.. you can ask, I don't think people would prefer to make postmerge private though. I think people prefer to maintain an open posture and keep things transparent.

at least there are cases where people need to goto the pipeline and find the wheel that is built for the commit.

Reply:
ok, let me ask around ;)